open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java

```diff
@@ -252,4 +256,25 @@ public Connection getPlatformSecurityConnection() throws OMAGNotAuthorizedExcept
 
         return restResult.getConnection();
     }
+
+    /**
+     * Return all the the OMAG Server configurations that are stored on this platform
+     *
+     * @return the OMAG Server configurations that are stored on this platform
+     * @throws OMAGNotAuthorizedException the supplied userId is not authorized to issue this command.
+     * @throws OMAGInvalidParameterException invalid parameter.
+     * @throws OMAGConfigurationErrorException unusual state in the admin server.
+     */
+    public Set<OMAGServerConfig> getPlatformServerConfigs() throws OMAGNotAuthorizedException,
```

Comment:
This is a confusing name since the server configurations are not stored in the platform.  They are managed by the config store connector.  This may be being shared by multiple platforms (for example when we run multiple servers in IntelliJ, all of the configuration is visible to all platforms).  A better name would be `getAllServerConfigurations()` since it spells out configuration to match the other methods in this module and does not imply that these configurations are tied to the platform.

Reply:
OK Changing